backend/app/__init__.py

```
@@ -1,32 +1,37 @@
     
+    # CORSの設定
+    CORS(app)
+    
     # データベースの初期化
     db.init_app(app)
-
+    migrate.init_app(app, db)
+    
     # モデルのインポート
     with app.app_context():
         from app.models import models
```

Ruff (0.8.2)

`app.models.models` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)